packages/polaris-viz/src/components/LineChartRelational/LineChartRelational.tsx

```diff
@@ -78,13 +83,7 @@ export function LineChartRelational(props: LineChartRelationalProps) {
           return (
             <Fragment>
               <MissingDataArea {...props} data={data} />
-              <RelatedAreas
-                data={data}
-                // remount the area otherwise it can't animate
```

Comment:
Why are we removing this comment?

Reply:
My bad it wasn't intentional, I'll add it back 👍